types.ts

```diff
@@ -0,0 +1,169 @@
+import {RedisClient, Callback, ClientOpts} from 'redis';
+import * as Route from 'route-parser';
+
+export interface IRedis extends RedisClient {
+  closing: boolean;
+  quitAsync(): Promise<void>;
+};
+export interface IRedisOptions extends ClientOpts {};
+
+// import * as IORedis from 'ioredis';
+// export interface IRedis extends IORedis.Redis {};
+// export type RedisOptions = IORedis.RedisOptions | string;
+
+export type ServiceConfig = {
+  environment: string;
+  hydra: HydraConfig;
+};
+export type ServiceDetails = {
+  serviceName: string;
+  serviceVersion?: string;
+  serviceIP?: string;
+  serviceDNS?: string;
+  servicePort?: any;
```

Comment:
why any?

Reply:
I was having some difficulty getting it to compile with a stricter requirement - I'll take another look now that I'm a little more familiar with TS, a union type will probably do the trick.